app/manager-frontend/src/components/agent/AgentSummary.jsx

```diff
@@ -0,0 +1,119 @@
+import {
+  Heading,
+  HStack,
+  Text,
+  LinkBox,
+  LinkOverlay,
+  Box,
+  Divider,
+  Center,
+  Stack,
+  useColorModeValue,
+  IconButton,
+  VStack,
+  Icon,
+} from "@chakra-ui/react";
+import { FaTrash, FaDesktop, FaDatabase } from "react-icons/fa";
+import { Link } from "react-router-dom";
+
+export const AgentSummary = ({
+  agentName,
+  href,
+  clientId,
+  dataSourceId,
+  onDelete,
+  ...p
+}) => (
+  <Center py={2}>
+    <LinkBox width="700px">
+      <Box
+        width="700px"
+        bg={useColorModeValue("white", "gray.900")}
+        boxShadow="lg"
+        rounded="md"
+        p={5}
+        overflow="hidden"
+        {...p}
+        _hover={{
+          transition: "0.3s",
+          transform: "translateY(-2px)",
+          boxShadow: "md",
+        }}
+      >
+        <LinkOverlay as={Link} to={`${href}`} />
```

Comment:
```suggestion
        <LinkOverlay as={Link} to={href} />
```
Since href is already a string.